src/document/position.js

```diff
@@ -5,7 +5,12 @@
 
 'use strict';
 
-CKEDITOR.define( [ 'utils', 'ckeditorerror' ], ( utils, CKEditorError ) => {
+CKEDITOR.define( [ 'document/rootelement', 'utils', 'ckeditorerror' ], ( RootElement, utils, CKEditorError ) => {
+	const SAME = 0;
+	const AFTER = 1;
+	const BEFORE = -1;
+	const DIFFERENT = -2;
```

Comment:
I did it so you can easily use `Array.prototype.sort` with `Position.compareWith`